### ISSUE TITLE
Android run fails. The SDK Build Tools revision (23.0.1) is too low...

### ISSUE THREAD

**Issue by lesharadkovich** [idehub/react-native-google-analytics-bridge#259]
### Environment

"react": "16.3.1",
"react-native": "https://github.com/expo/react-native/archive/sdk-27.0.0.tar.gz",
"react-native-google-analytics-bridge": "^5.8.0",

### Steps to Reproduce

Nothing special, just installed and linked this lib.

### Expected Behavior

Build is stable

### Actual Behavior

Trying to run in Android Studio and that what it says:
`The SDK Build Tools revision (23.0.1) is too low for project ':react-native-google-analytics-bridge'. Minimum required is 25.0.0`

Sorry if my question is very simple, I'm newbie in native android development

**Comment by cbrevik** [comment]
Hi. I see you’re using Expo. You cannot use native libs (like this one) with Expo

**Comment by lesharadkovich** [comment]
@cbrevik I've detached to ExpoKit, so I'm able to use native libraries

**Comment by CptFabulouso** [comment]
Everyone who upgrades to RN 0.56.0 will get this error, sdk versions now default to these values
```
    buildToolsVersion = "26.0.3"
    minSdkVersion = 16
    compileSdkVersion = 26
    targetSdkVersion = 26
    supportLibVersion = "26.1.0"
```
so targetVersion etc. in android/build.gradle need to be updated. [An example of react-native-config lib](https://github.com/luggit/react-native-config/pull/248)

**Comment by abartolo** [comment]
@cbrevik Is there a plan to update the project as @CptFabulouso  said for RN 0.56+ users?

**Comment by cbrevik** [comment]
Yes I'll look into this @abartolo. Been away travelling without Mac so hadn't had a chance yet.

**Comment by abartolo** [comment]
@cbrevik Awesome thank you! Just glad someone is looking into :)

**Comment by abartolo** [comment]
@cbrevik I created a PR https://github.com/idehub/react-native-google-analytics-bridge/pull/270 for the solution @CptFabulouso proposed. I was able to build and run the app successfully.

**Comment by cbrevik** [comment]
Hi, this should be fixed in version `5.9.0`
